### DISCUSSION TITLE
Can't get my Mojo working.

### DISCUSSION THREAD

**Discussion by JonkerKoorts** [modular/modular#3219] [Q&A] [answered]
I have done everything according to the setup guide on the docs of modular but still can't get mojo to work as a command: 

Last login: Thu Jul 11 20:19:08 on ttys007
jonkerkoorts@Jonkers-MacBook-Air ~ % curl -s https://get.modular.com | sh -
Updating Homebrew
==> Updating Homebrew...
Updated 2 taps (ngrok/ngrok and homebrew/cask).
==> New Casks
crashplan                                notchnook
==> Outdated Casks
ngrok

You have 1 outdated cask installed.
You can upgrade it with brew upgrade
or list it with brew outdated.
Installing/upgrading Modular using Homebrew
Warning: modularml/packages/modular 0.8.0 already installed
__  __           _       _
|  \/  | ___   __| |_   _| | __ _ _ __
| |\/| |/ _ \ / _` | | | | |/ _` | '__|
| |  | | (_) | (_| | |_| | | (_| | |
|_|  |_|\___/ \__,_|\__,_|_|\__,_|_|

Welcome to the Modular CLI!

To get started, see:
MAX  - https://modul.ar/install-max
Mojo - https://modul.ar/install-mojo

For info about this tool, type "modular --help".
jonkerkoorts@Jonkers-MacBook-Air ~ % python3 -m venv mojo-venv && source mojo-venv/bin/activate
(mojo-venv) jonkerkoorts@Jonkers-MacBook-Air ~ % modular install mojo
# Found release for https://packages.modular.com/mojo @ 24.4.0
# Downloading archive: packages/24.4.0/mojo-arm64-apple-darwin22.6.0-24.4.0-14-0.tar.zst
Downloaded           [ █████████████████ ] 100% 86.3MiB/86.3MiB               
# Extracting downloaded archives. 
Extracted            [ █████████████████ ] 100% 86.3MiB/86.3MiB               
# Running post-install scripts...
Installed            [ █████████████████ ] 100%                               

🔥 Mojo installed! 🔥

Mojo's Python virtual environment created at /Users/jonkerkoorts/.modular/pkg/packages.modular.com_mojo/venv

If you are using ZSH (default on macOS), run the following commands:

echo 'export MODULAR_HOME="/Users/jonkerkoorts/.modular"' >> ~/.zshrc
echo 'export PATH="/Users/jonkerkoorts/.modular/pkg/packages.modular.com_mojo/bin:$PATH"' >> ~/.zshrc
source ~/.zshrc

If you are using bash, run the following commands:

BASHRC=$( [ -f "$HOME/.bash_profile" ] && echo "$HOME/.bash_profile" || echo "$HOME/.bashrc" )
echo 'export MODULAR_HOME="/Users/jonkerkoorts/.modular"' >> "$BASHRC"
echo 'export PATH="/Users/jonkerkoorts/.modular/pkg/packages.modular.com_mojo/bin:$PATH"' >> "$BASHRC"
source "$BASHRC"

Then enter 'mojo' to start the Mojo REPL.

For tool help, enter 'mojo --help'.
For more docs, see https://docs.modular.com/mojo.

(mojo-venv) jonkerkoorts@Jonkers-MacBook-Air ~ % mojo --help
zsh: command not found: mojo
(mojo-venv) jonkerkoorts@Jonkers-MacBook-Air ~ % echo 'export MODULAR_HOME="/home/saket/.modular"' >> ~/.bashrc
echo 'export PATH="/home/saket/.modular/pkg/packages.modular.com_mojo/bin:$PATH"' >> ~/.bashrc
source ~/.bashrc
(mojo-venv) jonkerkoorts@Jonkers-MacBook-Air ~ % mojo --help
zsh: command not found: mojo
(mojo-venv) jonkerkoorts@Jonkers-MacBook-Air ~ %

**Comment by soraros** [comment, marked as answer]
You are using macOS, so likely the default shell is `zsh` (it's actually hinted):
> If you are using ZSH (default on macOS), run the following commands:
> ```sh
> echo 'export MODULAR_HOME="/Users/jonkerkoorts/.modular"' >> ~/.zshrc
> echo 'export PATH="/Users/jonkerkoorts/.modular/pkg/packages.modular.com_mojo/bin:$PATH"' >> ~/.zshrc
> source ~/.zshrc
> ```

**Reply by JonkerKoorts** [reply]
Thank you. It actually just took a restart of my os. 💪🔥

Do you have any recommendations on where to do a full course on Mojo? :)
